chutney/server/src/main/java/com/chutneytesting/execution/domain/schedule/CampaignScheduler.java

```diff
@@ -60,32 +62,31 @@ public void executeScheduledCampaigns() {
         }
     }
 
-    private Callable<Void> executeScheduledCampaignById(List<Long> campaignsId) {
+    private Callable<Void> executeScheduledCampaignById(Pair<List<CampaignExecutionRequest>, String> executionRequests) {
```

Comment:
We may rename this method. Where is the id ?
Maybe just `executeScheduledCampaigns`.